helm/charts/hpe-csi-driver/files/config.json

```diff
@@ -1,128 +1,405 @@
-        "severity": "critical",
-        "description": "product attribute recommended to be set to Server in /etc/multipath.conf",
-        "parameter": "product",
-        "recommendation": "\"Server\""
-    },
-    {
-        "category": "multipath",
-        "severity": "critical",
-        "description": "alua prioritizer is recommended. Can be set in /etc/multipath.conf",
-        "parameter": "prio",
-        "recommendation": "alua"
-    },
-    {
-        "category": "multipath",
-        "severity": "critical",
-        "description": "scsi_dh_alua device handler is recommended. Can be set in /etc/multipath.conf",
-        "parameter": "hardware_handler",
-        "recommendation": "\"1 alua\""
-    },
-    {
-        "category": "multipath",
-        "severity": "warning",
-        "description": "immediate failback setting is recommended. Can be set in /etc/multipath.conf",
-        "parameter": "failback",
-        "recommendation": "immediate"
-    },
-    {
-        "category": "multipath",
-        "severity": "critical",
-        "description": "immediately fail i/o on transient path failures to retry on other paths, value=1. Can be set in /etc/multipath.conf",
-        "parameter": "fast_io_fail_tmo",
-        "recommendation": "5"
-    },
-    {
-        "category": "multipath",
-        "severity": "critical",
-        "description": "queueing is recommended for 150 seconds, with no_path_retry value of 30. Can be set in /etc/multipath.conf",
-        "parameter": "no_path_retry",
-        "recommendation": "30"
-    },
-    {
-        "category": "multipath",
-        "severity": "warning",
-        "description": "service-time path selector is recommended. Can be set in /etc/multipath.conf",
-        "parameter": "path_selector",
-        "recommendation": "\"service-time 0\""
-    },
-    {
-        "category": "multipath",
-        "severity": "critical",
-        "description": "vendor attribute recommended to be set to Nimble in /etc/multipath.conf",
-        "parameter": "vendor",
-        "recommendation": "\"Nimble\""
-    },
-    {
-        "category": "multipath",
-        "severity": "critical",
-        "description": "group paths according to ALUA path priority of active/standby. Recommended to be set to group_by_prio in /etc/multipath.conf",
-        "parameter": "path_grouping_policy",
-        "recommendation": "group_by_prio"
-    },
-    {
-        "category": "multipath",
-        "severity": "critical",
-        "description": "tur path checker is recommended. Can be set in /etc/multipath.conf",
-        "parameter": "path_checker",
-        "recommendation": "tur"
-    },
-    {
-        "category": "multipath",
-        "severity": "critical",
-        "description": "infinite value is recommended for timeout in cases of device loss for FC. Can be set in /etc/multipath.conf",
-        "parameter": "dev_loss_tmo",
-        "recommendation": "infinity"
-    }
-]
\ No newline at end of file
+{
```

Comment:
should it be an array?

Reply:
Earlier we had common config for all the device and distros, hence it was a simple json array of all the configs .
Since now we have a detailed json structure with different sub section based on device type and distro , we have modified and generated a new json format.